pkg/filter/filter.go

```diff
@@ -172,7 +172,10 @@ func (f *Filter) shouldDiscardByBuiltInDDLAllowlist(ddlType timodel.ActionType)
 		timodel.ActionAddPrimaryKey,
 		timodel.ActionDropPrimaryKey,
 		timodel.ActionAddColumns,
-		timodel.ActionDropColumns:
+		timodel.ActionDropColumns,
```

Comment:
Please add a comment in here to tell that `timodel.ActionDropColumns` is removed in TiDB and will never be used after v6.2.0 or and add a ref link.